xmodule/capa/capa_problem.py

```diff
@@ -432,8 +442,12 @@ def grade_answers(self, answers):
 
         # if answers include File objects, convert them to filenames.
         self.student_answers = convert_files_to_filenames(answers)
-        new_cmap = self.get_grade_from_current_answers(answers)
+        if settings.FEATURES.get("ENABLE_GRADING_METHOD_IN_PROBLEMS", False):
+            new_cmap = self.get_grade_from_answers(answers)
```

Comment:
Why not just push down the dependence on the feature flag into `get_grade_from_answers()`? It's necessary there, but need not appear elsewhere, making the logic easier to follow.